dali/image/tiff_test.cc

```diff
@@ -19,6 +19,8 @@
 
 namespace dali {
 
+using legacy_impl::TiffBuffer;
```

Comment:
If `namespace legacy_impl` is cut out, this is also redundant

Reply:
gone